.github/ISSUE_TEMPLATE/transverse-issue.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Transverse issue
+about: An issue that impacts ALL reactor projects
```

Comment:
Perhaps we should be even more concrete here, because some may think that "Flux is not working" in an issue that impacts all reactor projects :D

Reply:
yeah, will reword